build.cake

```diff
@@ -9,7 +9,7 @@ var configuration = Argument("configuration", "Release");
 // TOOLS / ADDINS
 //////////////////////////////////////////////////////////////////////
 
-#addin "nuget:https://www.nuget.org/api/v2?package=MagicChunks&version=1.2.0.58"
+#addin "nuget:?package=MagicChunks"
```

Comment:
Please add version, because we got warning 

```
The 'tool' directive is attempting to install the 'MagicChunks' package
without specifying a package version number.
More information on this can be found at https://cakebuild.net/docs/tutorials/pinning-cake-version
It's not recommended, but you can explicitly override this warning
by configuring the Skip Package Version Check setting to true
(i.e. command line parameter "--settings_skippackageversioncheck=true",
environment variable "CAKE_SETTINGS_SKIPPACKAGEVERSIONCHECK=true",
read more about configuration at https://cakebuild.net/docs/fundamentals/configuration)
```